### PR TITLE
Force-driven inlet

### PR DESCRIPTION
**Pull request by GuillermoCasas**
A force-driven inlet has been implemented. itr works fine with DEM. All the DEM tests work.

### CODE REVIEW

**Comment by joaquinirazabal** [comment]
Nice implementation, this would be useful.
This makes me think about creating a specific benchmarks to check if the inlet is working correctly (I don't remember any).
Now I am working in my PhD thesis, but I can do it in the future

**Comment by GuillermoCasas** [comment]
@joaquinirazabal Sure, we need some benchmarks. I think the issue of inlets is quite interesting, full of technicalities. 

I would like to revisit the concept after my Thesis too and see what work can be done/written about. I am sure Miguel Angel too. Let us talk, when we are both done. Regards

File: applications/DEM_application/custom_conditions/RigidFace.h [hunk 1]
```diff
@@ -1,13 +1,8 @@
-//
 // Author: Miquel Santasusana msantasusana@cimne.upc.edu
```
Comment (maceligueta) [inline applications/DEM_application/custom_conditions/RigidFace.h]: @GuillermoCasas  should be author here, too.

**Review by maceligueta — CHANGES_REQUESTED** [review]
I see a lot of code duplicated with PR #291 (the same changes are in both branches).
I propose to finish PR #291, then merge master to this branch, then review the code.

From Guillermo: I have reviewed all your comments in the previous pull request, which I will remove and apply all changes to this pull request.

File: applications/DEM_application/custom_conditions/RigidFace.h (hunk 1 above)
Comment (GuillermoCasas) [reply applications/DEM_application/custom_conditions/RigidFace.h]: Ok!

**Review by maceligueta — CHANGES_REQUESTED** [review]
TimesPartialRelease still to be removed

Ok, I will erase it

**Review by maceligueta — APPROVED** [review]

**Pull request merged**